ebuild-writing/eapi/text.xml

```diff
@@ -616,6 +616,774 @@ installation targets.
+      </li>
+      <li>
+        <p>
+        <c>mapfile</c> (AKA <c>readarray</c>) now accepts a delimiter via
+        <c>-d</c>, with a <c>-t</c> option to strip it from read data
+        (bash 4.4+). The two following solutions to grab output from
+        <c>find(1)</c> are now equivalent:
+        </p>
+
+<pre>
+# old solution
+local x files=()
+while read -d '' -r x; do
+	files+=( "${x}" )
+done &lt; &lt;(find -print0)
+
+# new solution
+local files=()
+mapfile -d '' -t files &lt; &lt;(find -print0)
+</pre>
+
+      </li>
+      <li>
+        <p>
+        A new set of transformations is available via <c>${foo@...}</c>
+        parameter expansion (4.4+), e.g. to print a value with necessary
+        quoting:
+        </p>
+
+<pre>
+$ var="foo 'bar' baz"
+$ echo "${var@Q}"
+'foo '\''bar'\'' baz'
+</pre>
+
+        <p>
+        For more details, see: <c>info bash</c> or the
+        <uri link="https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html">
+        Bash reference manual</uri>.
+        </p>
+      </li>
+      <li>
+        <p>
+        <c>local -</c> can be used to limit single-letter (mangled via
+        <c>set</c>) shell option changes to the scope of the function, and
+        restore them upon returning from it (4.4+). The following two functions
+        are now equivalent:
+        </p>
+
+<pre>
+# old solution
+func() {
+	local prev_shopt=$(shopt -p -o noglob)
+	set -o noglob
+	${prev_shopt}
+}
+
+# new solution
+func() {
+	local -
+	set -o noglob
+}
+</pre>
+
+        <p>
+        The complete information on all changes and new features can be found
+        in the
+        <uri link="https://git.savannah.gnu.org/cgit/bash.git/tree/NEWS?h=bash-5.0">
+        bash 5.0 (and earlier) release notes</uri>.
+        </p>
+      </li>
+    </ul>
+  </dd>
+</dl>
+
+</body>
+</subsection>
+<subsection>
+<title>EAPI 8 variables</title>
+<body>
+
+<dl>
+  <dt>Selective fetch/mirror restriction</dt>
+  <dd>
+    <p>
+    Before EAPI 8, fetch and mirror restrictions applied globally. That is,
+    if you needed to apply the respective restriction to at least one distfile,
+    you had to apply it to them all. However, sometimes packages used a
+    combination of proprietary and free distfiles, the latter including e.g.
+    third party patches, artwork. They had to be mirror-restricted so far.
+    </p>
+
+    <p>
+    EAPI 8 brings the possibility of undoing fetch and mirror restriction
+    for individual files. The rough idea is that you put a <c>RESTRICT</c>
+    as before, then use the special <c>fetch+</c> prefix to specify URLs that
+    can be fetched from, or <c>mirror+</c> prefix to reenable mirroring of
+    individual files.
+    </p>
+
+    <p>
+    Similarly to how the <c>fetch</c> restriction implies a <c>mirror</c>
+    restriction, the <c>mirror</c> override implies a <c>fetch</c> override.
+    This might sound confusing at first but when you think about it, it's
+    perfectly logical.
```

Comment:
```suggestion
```

Reply:
Not sure what you mean, the suggested change would result in an incomplete sentence.

I have removed the whole last sentence of the paragraph.

Reply:
This was Github's UI being confusing. I wanted the whole last sentence gone (which is what's happened), sorry!